sqlparser/parse_test.go

```diff
@@ -211,6 +211,20 @@ var (
 		output: "select /* string table alias */ 1 from t as 't1'",
 		// mysql allow to use single quote for column/table aliases
 		dialect: mysql.NewMySQLDialect(),
+	}, {
+		input: `select * from mytable where "AGE" = 1 and "TEST" = 'test'`,
+		// postgres allow to use double quote string for columns
+		dialect: postgresql.NewPostgreSQLDialect(),
+	}, {
+		input:  `select * from mytable where "test" = "test"`,
+		output: `select * from mytable where 'test' = 'test'`,
+	}, {
+		input:  `select * from mytable where "test" = 1 and 'value' = 'value'`,
+		output: `select * from mytable where 'test' = 1 and 'value' = 'value'`,
+	}, {
+		input:   `SELECT "id", "landline_number" AS "landlineNumber", "removal" FROM "users" AS "User" where "User"."is_active"`,
```

Comment:
lets add some somparison too: `WHERE "User"."AGE" = 123` and simple `"AGE"='123'` without table name.

Reply:
plus I remember that you mentioned some integration tests with double quotes. Can we extend them with additional cases that we caught?